src/Aspire.Hosting/ApplicationModel/EnvironmentCallbackContext.cs

```diff
@@ -10,6 +10,12 @@ namespace Aspire.Hosting.ApplicationModel;
 /// <param name="environmentVariables">The environment variables associated with this execution.</param>
 public class EnvironmentCallbackContext(DistributedApplicationExecutionContext executionContext, Dictionary<string, string>? environmentVariables = null)
 {
+    /// <summary>
+    /// Obsolete. Use ExecutionContext instead. Will be removed in next preview.
+    /// </summary>
+    [Obsolete("Use ExecutionContext instead")]
+    public string PublisherName => ExecutionContext.Operation == DistributedApplicationOperation.Publish ? "manifest" : "dcp";
```

Comment:
```suggestion
    public string PublisherName => ExecutionContext.Operation == DistributedApplicationOperation.Publish ? "manifest" : null;
```

I would make it null, not dcp (was it dcp before?)

Reply:
It is "dcp".

Reply:
Cool